README.md

```diff
@@ -108,18 +108,18 @@ your new changes you can see the `(BUILD).tags` file in the `tmp` directory.
 
 ### PHP
 
-In this example adding PHP 7.3-rc for cli and fpm:
+In this example adding PHP 7.4-rc for cli and fpm:
 
 ```diff
 build-cli: clean-tags
-	./build-php.sh cli 7.3 3.9
-	./build-php.sh cli 7.3 3.10
-+	./build-php.sh cli 7.3-rc 3.10
+	./build-php.sh cli 7.4 3.10
+	./build-php.sh cli 7.4 3.11
++	./build-php.sh cli 7.4-rc 3.12
```

Comment:
Giving the current next PHP release is 8, doesn't it make more sense to use 8.0 here, or even 8.1?

Reply:
Once we add the PHP 8.0 images, we can update this documentation. I'm just updating it now, to reflect our current latest state for people using these images :)